client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -202,6 +206,20 @@ static StreamManager create(ClientConfig clientConfig) {
      */
     boolean deleteScopeRecursive(String scopeName) throws DeleteScopeFailedException;
 
+    /**
+     * Re-read an event that was previously read, by passing the pointer returned from
+     * {@link EventRead#getEventPointer()}.
+     * This does not affect the current position of the reader.
+     * <p>
+     * This is a non-blocking call. Passing invalid offsets will throw exception.
```

Comment:
The user isn't passing an offset here. It's an event pointer, which is an object we created. So this is not a concern. Instead you should specify what happens if the stream has been deleted or the data truncated away.